packages/server-wallet/src/protocols/direct-funder.ts

```diff
@@ -59,12 +58,15 @@ export class DirectFunder {
     // otherwise, deposit
     const amountToDeposit = BN.sub(targetAfter, currentAmount); // previous checks imply this is >0
     await ChainServiceRequest.insertOrUpdate(channel.channelId, 'fund', tx);
-    await this.chainService.fundChannel({
-      channelId: channel.channelId,
-      assetHolderAddress: assetHolderAddress,
-      expectedHeld: BN.from(currentAmount),
-      amount: BN.from(amountToDeposit),
-    });
+    await response.queueChainRequest([
```

Comment:
```suggestion
    response.queueChainRequest([
```

Reply:
Yup! 

Unfortunately, I can't seem to find any lint rule to catch unecessary awaits :(